app/views/json_submissions/schema_v1.json.erb

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "<%= json_submissions_schema_v1_url %>",
+  "title": "Form submission - version 1",
+  "type": "object",
+  "properties": {
+    "form_name": {
+      "description": "The name of the form",
```

Comment:
@hannahkc would you be able to look at the descriptions in this file, or nominate another content designer to? Happy to jump on a call and explain what they are if it isn't clear. Also no rush to look at this right away.